processing/src/main/java/org/apache/druid/segment/virtual/ExpressionSelectors.java

```diff
@@ -108,7 +108,11 @@ public Object getObject()
       {
         // No need for null check on getObject() since baseSelector impls will never return null.
         //noinspection ConstantConditions
```

Comment:
Is the `//noinspection` still on the right line?

Reply:
I don't believe it is needed anymore, will remove.